data/ecosystem/vendors.yaml

```diff
@@ -270,9 +270,9 @@
   contact: hello@openobserve.ai
   oss: true
   commercial: true
-- name: OpenText
+- name: OpenText Application Observability
   nativeOTLP: true
-  url: https://docs.microfocus.com/doc/OBS/SaaS/ConfigureOTELCollector
+  url: https://www.opentext.com/products/core-application-observability
```

Comment:
Can you point this to a page where we can verify OTLP support?

Reply:
For example here:
- https://community.opentext.com/it_ops_mgt/b/it-ops-cloud-blog/posts/what-s-new-in-operations-bridge-ce-24-3
A quote:

> "After our first Application Observability 24.2 release (for details, refer to OpenText Application Observability) is now available - a technical overview!](https://community.opentext.com/it_ops_mgt/b/it-ops-cloud-blog/posts/opentext-application-observability-is-now-available-_2d00_-a-technical-overview)), we have further extended this solution by offering the following capabilities:
> 
> Get OTel data from JavaScript and other applications via the additional OTLP receiver
> Previous versions of Application Observability only supported ingesting telemetry data through OTLP/gRPC. With this release, Application Observability supports ingesting OTel logs, traces and metrics over the OpenTelemetry Protocol (OTLP) as well (through HTTP/Protobuf on port 4318). This enhancement allows you to receive the data from JavaScript browser-based applications.
> 
> Data ingestion through OTLP/gRPC has also changed from the port 443 to the port 4317"